backend/src/test/java/gov/cdc/usds/simplereport/api/QueueManagementTest.java

```diff
@@ -71,17 +71,19 @@ public void updateItemInQueue() throws Exception {
         String orderId = o.getInternalId().toString();
         DeviceType d = _dataFactory.getGenericDevice();
         String deviceId = d.getInternalId().toString();
+        String dateTested = "2020-12-31T14:30:30Z";
         ObjectNode variables = JsonNodeFactory.instance.objectNode()
                 .put("id", orderId)
                 .put("deviceId", deviceId)
-                .put("result", TestResult.POSITIVE.toString());
+                .put("result", TestResult.POSITIVE.toString())
+                .put("dateTested", dateTested);
 
         performQueueUpdateMutation(variables);
-
         TestOrder updatedTestOrder = _testOrderService.getTestOrder(orderId);
         assertEquals(updatedTestOrder.getDeviceType().getInternalId().toString(), deviceId);
         assertEquals(updatedTestOrder.getTestResult(), TestResult.POSITIVE);
         assertEquals(updatedTestOrder.getTestEvent(), null);
+        assertEquals(updatedTestOrder.getDateTested().toInstant().toString(), dateTested);
```

Comment:
This should have failed without your TZ modification. It doesn't seem to. I'm so confused...